### PR TITLE
Client base setup

### PR DESCRIPTION
**Pull request by OmiCoding**
Added theme.
Changed Registration, Login Page.
Created base setup for client side.
Created Initial launch page.
Added loggedIn prop to check user is logged in.

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -1,69 +1 @@
-const deleteBtn = document.querySelectorAll('.del')
```
Comment (FullStackCorm) [inline public/js/main.js:1]: holup i'm really tired, we still need all of this don't we?

File: routes/homeRoutes.js [hunk 2]
```diff
@@ -12,6 +12,6 @@ router.post("/signup", authController.postSignup);
 router.get("/signup", authController.getSignup);
 router.get("/login", authController.getLogin);
 router.post("/login", authController.postLogin);
-router.post("/logout", authController.logout);
+router.get("/logout", authController.logout);
```
Comment (FullStackCorm) [inline routes/homeRoutes.js:15]: shouldn't this be a post?

**Review by FullStackCorm — CHANGES_REQUESTED** [review]
review public/js/main.js and routes/homeRoutes.js

**Review by HermanDhillon — APPROVED** [review]
Looks good to me!

File: public/js/main.js (hunk 1 above)
Comment (HermanDhillon) [reply public/js/main.js:1]: All of the previous content in Main.js was just todo list stuff copied in from Leons code, we haven't used it.

File: routes/homeRoutes.js (hunk 2 above)
Comment (OmiCoding) [reply routes/homeRoutes.js:15]: Fetch uses ajax so it doesn't allow you to redirect through the backend. We could redirect client side, but not sure what the team wants in how to redirect the user.

Comment (HermanDhillon) [reply routes/homeRoutes.js:15]: An issue for this has been created and will be resolved later.

**Pull request merged**